docs/source/teams/data_lens.rst

```diff
@@ -594,12 +594,350 @@ ______________________________
 This section provides example Data Lens connectors for various popular data
 sources.
 
+.. _data-lens-databricks:
+
+Databricks
+----------
+
+Below is an example of a Data Lens connector for Databricks. This example uses
+a schema consistent with the Berkeley DeepDrive dataset format.
+
+.. code-block:: python
+    :linenos:
```

Comment:
This is a great snippet - along with google big query both are standalone operators. Maybe we should add them as standalone files somewhere, maybe tutorial repo or even inside fiftyone repo itself?